custom_components/tuya_local/devices/diivoo_dwv010.yaml

```diff
@@ -25,22 +25,46 @@ secondary_entities:
         type: string
         name: operation
   - entity: sensor
-    name: Water use
     class: volume
+    name: Cumulative water usage
     category: diagnostic
     dps:
       - id: 112
         type: integer
         name: sensor
         unit: "l"
+  - entity: sensor
+    class: volume
+    name: Daily water usage
```

Comment:
This can just be an attribute of the water use sensor (unrenamed).  There doesn't seem to be a lot of value in having two separate sensors showing different versions of the same measurement.
DP 112 would benefit from having `class: total_increasing` added, so it can be added to the energy dashboard where all the time derived versions can be automatically available from Home Assistant's own data recording.